Tasks/count.js

```diff
@@ -1,17 +1,16 @@
 // Sum all number values in dict
+'use strict';
 
-count = (obj) => {
-  sum = 0;
-  {
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
+const sumOfNumbers = (obj) => {
+  let result = 0;
+
+  const keys = Object.keys(obj);
+
+  keys.map((key) => {
```

Comment:
Use `map` only to create outgoing array as a result of `map` call but not to change values in outer scope